### PR TITLE
docs: add FS storage to docs

### PR DESCRIPTION
**Pull request by cbaker6**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### Issue Description

Readme is missing FS adapter even though link to full docs has details about the FS adapter.

Related issue: #6768 and https://github.com/parse-community/docs/pull/780

### Approach

Add the FS adapter to the README and clean up this section to match the rest of the document.

### TODOs before merging

- [x] Add changes to documentation (guides, repository pages, in-code descriptions)
- [x] A changelog entry is created automatically using the pull request title (do not manually add a changelog entry)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -489,11 +489,12 @@ You can also find more adapters maintained by the community by searching on [npm
...
-* `GridFSBucketAdapter`, which is backed by MongoDB;
-* `S3Adapter`, which is backed by [Amazon S3](https://aws.amazon.com/s3/); or
-* `GCSAdapter`, which is backed by [Google Cloud Storage](https://cloud.google.com/storage/)
+* `GridFSBucketAdapter` - which is backed by MongoDB
+* `S3Adapter` - which is backed by [Amazon S3](https://aws.amazon.com/s3/)
+* `GCSAdapter` - which is backed by [Google Cloud Storage](https://cloud.google.com/storage/)
+* `FSAdapter` - local file storage
 
-`GridFSBucketAdapter` is used by default and requires no setup, but if you're interested in using S3 or Google Cloud Storage, additional configuration information is available in the [Parse Server guide](http://docs.parseplatform.org/parse-server/guide/#configuring-file-adapters).
+`GridFSBucketAdapter` is used by default and requires no setup, but if you're interested in using S3, Google Cloud Storage, or local file storage additional configuration information is available in the [Parse Server guide](http://docs.parseplatform.org/parse-server/guide/#configuring-file-adapters).
```
Comment (mtrezza) [inline README.md]: To be fair, could you add the company name to S3?

> using Amazon S3, Google Cloud Storage,

Comment (cbaker6) [reply README.md]: Done

**Review by mtrezza — APPROVED** [review]
Looks good, is this ready for merge?

**Comment by cbaker6** [comment]
Yup, it’s ready

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-alpha.6](https://github.com/parse-community/parse-server/releases/tag/5.0.0-alpha.6)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-beta.10](https://github.com/parse-community/parse-server/releases/tag/5.0.0-beta.10)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0](https://github.com/parse-community/parse-server/releases/tag/5.1.0)

**Pull request merged**